x-pack/plugins/cases/tsconfig.json

```diff
@@ -52,6 +52,7 @@
     "@kbn/safer-lodash-set",
     "@kbn/logging-mocks",
     "@kbn/ecs",
+    "@kbn/core-saved-objects-base-server-mocks",
```

Comment:
I think this is so we can get the mock for the serializer? I didn't add it explicitly, maybe vs code did 🤷‍♂️